randovania/games/samus_returns/logic_database/Surface - East.txt

```diff
@@ -791,9 +836,10 @@ Extra - asset_id: collision_camera_018
       All of the following:
           Morph Ball and Shoot Any Missile
           Any of the following:
+              # Get to the Door
```

Comment:
DB shows `Impossible` as a path. Idk what that's supposed to be for.

Reply:
The DB doesn't show impossible, the pretty-printer does:
![grafik](https://github.com/randovania/randovania/assets/38186597/c6188409-00e6-4aab-8c0e-752ac9939d30)